packages/reporter/src/hooks/hooks.tsx

```diff
@@ -30,18 +29,10 @@ export interface HookOpenInIDEProps {
 }
 
 const HookOpenInIDE = ({ invocationDetails }: HookOpenInIDEProps) => {
-  if ('__vite__' in window) {
-    return (
-      <OpenFileInIDE fileDetails={invocationDetails} className='hook-open-in-ide'>
-        <OpenIcon viewBox="0 0 16 16" width="12" height="12" /> <span>Open in IDE</span>
-      </OpenFileInIDE>
-    )
-  }
-
   return (
-    <FileOpener fileDetails={invocationDetails} className='hook-open-in-ide'>
+    <OpenFileInIDE fileDetails={invocationDetails} className='hook-open-in-ide'>
       <OpenIcon viewBox="0 0 16 16" width="12" height="12" /> <span>Open in IDE</span>
-    </FileOpener>
+    </OpenFileInIDE>
```

Comment:
The component for the hook is a little unique, in that it does not need an inner link and has a unique icon. It makes more sense to render OpenFileInIDE directly rather than make FileOpener more complex.